dev/package.json

```diff
@@ -1,15 +1,15 @@
 {
     "name": "framer-motion--dev",
-    "version": "8.5.1",
+    "version": "8.5.2-alpha.4",
```

Comment:
This needs to be non-alpha before we merge, right?

Reply:
Before publish but not so fussed about merge, normally the version bumps happen on `main`

Reply:
👌🏻